README_SECURITYCONTEXT.md

```diff
@@ -0,0 +1,18 @@
+# Security Context
+
+The default `container-security-context` check checks the `SecurityContext`
+for 
+
+* Containers with writeable root filesystems
+* Containers that run with user ID or group ID < 10000
+* Privileged containers
+
+If you do not want all of this checks you can disable `container-security-context`
+and enable one or more of the following optional checks:
+
+* `container-security-context-user-group-id`
+* `container-security-context-privileged`
+* `container-security-context-readonlyrootfilesystem`
+
+In future releases the `container-security-context` will become *optional*
+and replaced by the more detailed checks.
```

Comment:
I'd like to commit to the following timeline, and think that it would be a good idea to mention it in this document.

* v1.10: Introduce the three new checks.
* v1.11: Make `container-security-context` optional, and make the three new checks run by default.
* v1.12: Remove `container-security-context`.

Since this is going to be a breaking change, I'd like to be as clear as possible with what's going to happen, so that users get a chance to prepare.

Reply:
I have updated the document